vagrant/provisioners/install-lucee.sh

```diff
@@ -63,7 +63,7 @@ sudo /bin/cp -f /vagrant/configs/lucee-server.xml /opt/lucee/lib/lucee-server/co
 sudo cp /vagrant/configs/server.xml /opt/lucee/tomcat/conf/server.xml
 
 sudo mkdir -p /opt/lucee/config/web
-sudo cp -f /vagrant/configs/web.xml /opt/lucee/tomcat/conf/web.xml
```

Comment:
The Lucee updated patch doesn't seem to work. I get a whole stack of Tomcat dumps. The CommandBox update seems to work just fine.